### ISSUE TITLE
Doc is missing one key instruction

### ISSUE THREAD

**Issue by gargleblaster** [meteorhacks/npm#75]
I was stuck with errors like `ReferenceError: module is not defined` until I stumbled upon the need to delete the node_modules directory.  I would recommend that you put that instruction into the README (in bold, italic, red, large text ;).

**Comment by arunoda** [comment]
I didn't get this.
How you get this?

we don't need a node_modules directory and you should not install npm
modules via npm.

**Comment by gargleblaster** [comment]
Being (somewhat) accustomed to Node but new to Meteor, I had already run `npm install` on a couple of modules, not realizing that meteorhacks:npm does its own install.  By the time I found your project and tried adding it to my app, I already had a node_modules directory, and nothing in the docs told me that it shouldn't be there or that meteorhacks:npm would install its own copy in a different location.  That's what I'm saying needs to be added to the README.

**Comment by wildeyes** [comment]
I agree @arunoda, it would be helpful to mention it, since meteorhacks:npm takes a different approach from what node developers are used to.

**Comment by afang** [comment]
+1 on this
